### PR TITLE
Add compat data for input-submit

### PR DESCRIPTION
**Pull request by maboa**
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/submit

### CODE REVIEW

File: html/elements/input/submit.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      "input": {
+        "input-submit": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/html/Element/input/submit",
+            "support": {
+              "chrome": {
+                "version_added": "1"
+              },
+              "chrome_android": {
+                "version_added": null
```
Comment (ddbeck) [inline html/elements/input/submit.json]: This is definitely `true`

File: html/elements/input/submit.json [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+              },
+              "edge": {
+                "version_added": null
```
Comment (ddbeck) [inline html/elements/input/submit.json]: Also `true`

File: html/elements/input/submit.json [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+              },
+              "edge_mobile": {
+                "version_added": null
```
Comment (ddbeck) [inline html/elements/input/submit.json]: Another `true` value

File: html/elements/input/submit.json [hunk 4]
```diff
@@ -0,0 +1,58 @@
+{
+  "html": {
+    "elements": {
+      "input": {
+        "input-submit": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/html/Element/input/submit",
```
Comment (ddbeck) [inline html/elements/input/submit.json]: Needs `"description": "<code>type=\"submit\"</code>"`

File: html/elements/input/submit.json [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+              },
+              "edge": {
+                "version_added": null
+              },
+              "edge_mobile": {
+                "version_added": null
+              },
+              "firefox": {
+                "version_added": "1",
+                "notes": "Unlike other browsers, Firefox by default <a href='http://stackoverflow.com/questions/5985839/bug-with-firefox-disabled-attribute-of-input-not-resetting-when-refreshing'>persists the dynamic disabled state</a> of a <code><button></code> across page loads. Use the <code><a href='https://developer.mozilla.org/docs/Web/HTML/Element/button#attr-autocomplete'>autocomplete</a></code> attribute to control this feature."
```
Comment (ddbeck) [inline html/elements/input/submit.json]: `<button>` needs to be escaped here (i.e., `&lt;button&gt;`)

File: html/elements/input/submit.json [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+              },
+              "firefox_android": {
+                "version_added": "4",
+                "notes": "Unlike other browsers, Firefox by default <a href='http://stackoverflow.com/questions/5985839/bug-with-firefox-disabled-attribute-of-input-not-resetting-when-refreshing'>persists the dynamic disabled state</a> of a <code><button></code> across page loads. Use the <code><a href='https://developer.mozilla.org/docs/Web/HTML/Element/button#attr-autocomplete'>autocomplete</a></code> attribute to control this feature."
```
Comment (ddbeck) [inline html/elements/input/submit.json]: `<button>` needs to be escaped here (i.e., `&lt;button&gt;`)

File: html/elements/input/submit.json (hunk 1 above)
Comment (maboa) [reply html/elements/input/submit.json]: :+1:

**Comment by ddbeck** [comment]
Excellent, thank you @maboa!

**Pull request merged**